llama.h

```diff
@@ -390,8 +390,9 @@ extern "C" {
     LLAMA_API uint32_t llama_n_ubatch   (const struct llama_context * ctx);
     LLAMA_API uint32_t llama_n_seq_max  (const struct llama_context * ctx);
 
-    LLAMA_API enum llama_vocab_type llama_vocab_type(const struct llama_model * model);
-    LLAMA_API enum llama_rope_type  llama_rope_type (const struct llama_model * model);
+    LLAMA_API enum llama_vocab_type   llama_vocab_type  (const struct llama_model   * model);
+    LLAMA_API enum llama_rope_type    llama_rope_type   (const struct llama_model   * model);
+    LLAMA_API enum llama_pooling_type llama_pooling_type(const struct llama_context * model);
```

Comment:
Move up next to the other `llama_context` functions

```suggestion
    LLAMA_API enum llama_pooling_type llama_pooling_type(const struct llama_context * ctx);
```

Reply:
Fixed the argument name. Moved up higher, but didn't seem like aligning signature was worth it.